certified-connectors/SignHost/Readme.md

```diff
@@ -0,0 +1,22 @@
+# SignHost
```

Comment:
Can you please update readme.md file as per the below template?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md